packages/styles/scss/patterns/blocks/callout-quote/index.scss

```diff
@@ -8,3 +8,15 @@
 @import '../../../globals/imports';
 @import '../../../internal/callout/callout';
 @import '../../../components/quote/quote';
+@import '../../../globals/utils/hang';
+
+@mixin callout-quote {
+  .#{$prefix}--callout-quote {
+    @include hang;
```

Comment:
@greeshmareddyg since the `sm` breakpoint is working as expected, I think this change should be inside a `carbon--breakpoint` content block.